brain/synapses.cpp

```diff
@@ -277,8 +281,8 @@ struct Synapses::Impl
             return;
 
         lunchbox::ScopedWrite mutex( _lock );
-        _loadAttributes( _afferent ? _postGIDs : _preGIDs,
-                         _afferent ? _preGIDs : _postGIDs );
+        _loadAttributes( _afferent ? _gids : _filterGIDs,
+                         _afferent ? _filterGIDs : _gids );
```

Comment:
This doesn't look right to me. Don't you have to call _loadAttributes( _gids, _filterGIDs ) regardless of the value of _afferent?